openfl/component/assigner/random_grouped_assigner.py

```diff
@@ -47,9 +47,11 @@ def define_task_assignments(self):
             'Task group percentages must sum to 100%')
 
         # Start by finding all of the tasks in all specified groups
-        self.all_tasks_in_groups = list(
-            set([task for group in self.task_groups for task in group['tasks']])
-        )
+        self.all_tasks_in_groups = list({
```

Comment:
Why 'list' is available here?

Reply:
There is not a list comprehension, it is only converting `set` to `list`